### PR TITLE
License expression display in Gallery

### PR DESCRIPTION
**Pull request by agr**
Addresses NuGet/NuGetGallery#6721

Will update PR once https://github.com/NuGet/ServerCommon/pull/234 is merged and non-private builds of a new library are available.

image

Worse case:

image

Should we put `<nobr>` (or set up css accordingly) around license names, so they are not split between lines?

### CODE REVIEW

**Comment by joelverhagen** [comment]
Screenshots please

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml [hunk 1]
```diff
@@ -146,6 +155,10 @@
     </div>
 }
 
+@* The following two helpers must be on a single line each so no extra whitespace is introduced in the expression when rendered. *@
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: These are only used in one place each and look pretty simple. Maybe consider just inlining them?

File: src/NuGetGallery/Controllers/PackagesController.cs [hunk 2]
```diff
@@ -624,6 +628,21 @@ public virtual async Task<ActionResult> DisplayPackage(string id, string version
 
             model.ReadMeHtml = await _readMeService.GetReadMeHtmlAsync(package);
 
+            if (!string.IsNullOrWhiteSpace(package.LicenseExpression))
```
Comment (scottbommarito) [inline src/NuGetGallery/Controllers/PackagesController.cs]: Add tests that verify that the license expression is split if it exists?

Comment (agr) [reply src/NuGetGallery/Controllers/PackagesController.cs]: Added

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml (hunk 1 above)
Comment (scottbommarito) [reply src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: Sounds good. Maybe add that comment in there?

**Pull request merged**